_studio/shared/src/libmfx_allocator_vaapi.cpp

```diff
@@ -619,14 +625,15 @@ mfxStatus mfxDefaultAllocatorVAAPI::SetFrameData(const VAImage &va_image, mfxU32
 
 #if (MFX_VERSION >= 1031)
     case VA_FOURCC_Y416:
-        if (mfx_fourcc != va_image.format.fourcc) return MFX_ERR_LOCK_MEMORY;
-
+    case VA_FOURCC_Y412:
+        if (mfx_fourcc == MFX_FOURCC_Y416)
```

Comment:
Need also add MFX_FOURCC_Y412?

Reply:
No. There isn't `MFX_FOURCC_Y412`, Here add mapping between  VA_FOURCC_Y412 VA Image  and `MFX_FOURCC_Y416` mfx surface

Reply:
Thanks for clarification.